code/modules/mob/living/simple_animal/hostile/human/frontiersman.dm

```diff
@@ -33,6 +33,19 @@
 	mob_spawner = /obj/effect/mob_spawn/human/corpse/frontier/ranged
 	r_hand = /obj/item/gun/ballistic/automatic/pistol/mauler/regular
 
+/mob/living/simple_animal/hostile/human/frontier/ranged/space
+	icon_state = "frontiersmanranged_mask"
```

Comment:
these guys need different icon states. just paste the forward dir of the hardsuit over